business_requirement/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Business Requirement
+====================
+
+This module is part of a set ("Business Requirement") and provides the basic models for business analysis and time estimation.
+
+Installation
+============
+
+No specific steps required
+
+
+
+Configuration
+=============
+
+No configuration steps required
+
+
+
+Usage
+=====
+
```

Comment:
As a guideline, here I expect to see steps that guide me to evaluate the feature, after I click on the Try Me button.